docs/en/advanced/Namespace-Management.md

```diff
@@ -317,20 +315,20 @@ Active sync also tries to avoid syncing when the target directory is heavily use
 It tries to look for a quiet period in UFS activity to start syncing between the UFS and the Alluxio space, to avoid overloading the UFS when it is busy.
 There are two configuration options that control this behavior.
 
-`alluxio.master.activesync.maxactivity` is the maximum number of activities in the UFS directory. 
+`alluxio.master.active.sync.max.activities` is the maximum number of activities in the UFS directory. 
 Activity is a heuristic based on the exponential moving average of number of events in a directory.
 For example, if a directory had 100, 10, 1 event in the past three intervals. 
 Its activity would be `100/10*10 + 10/10 + 1 = 3` 
-`alluxio.master.activesync.maxage` is the maximum number of intervals we will wait before synchronizing the UFS and the Alluxio space.
+`alluxio.master.active.sync.max.age` is the maximum number of intervals we will wait before synchronizing the UFS and the Alluxio space.
 
 The system guarantees that we will start syncing a directory if it is "quiet", or it has not been synced for a long period (period longer than the max age).
 
 For example, the following setting 
 
 ```
-alluxio.master.activesync.interval=30secs
-alluxio.master.activesync.maxactivity=100
-alluxio.master.activesync.maxage=5
+alluxio.master.active.sync.interval=30secs
+alluxio.master.ufs.active.sync.max.activities=100
```

Comment:
Why are some of the configurations prefixed with `ufs` and others not?

Reply:
oops, fixed. thanks.